datafusion-common/src/dfschema.rs

```diff
@@ -222,6 +222,16 @@ impl DFSchema {
         }
     }
 
+    pub fn fields_with_qualified(&self, qualifier: &str) -> Vec<&DFField> {
```

Comment:
It would be nice to add a docstring (`/// comment`) to this function explaining what it does